src/world/area_hos/hos_05/A2AAC0.c

```diff
@@ -196,13 +198,50 @@ ApiStatus func_80240E30_A2B070(Evt* script, s32 isInitialCall) {
 }
 
 // adjusts properties of EmitterVolume:GoldShimmer2 effect
-INCLUDE_ASM(s32, "world/area_hos/hos_05/A2AAC0", func_80240E50_A2B090);
+ApiStatus func_80240E50_A2B090(Evt* script, s32 isInitialCall) {
+    Bytecode* args = script->ptrReadPos;
+    EffectInstance* effect = (EffectInstance*) evt_get_variable(script, *args++);
+    s32 subtype = evt_get_variable(script, *args++);
+    s32 posX = evt_get_float_variable(script, *args++);
+    s32 posY = evt_get_float_variable(script, *args++);
+    s32 posZ = evt_get_float_variable(script, *args++);
+
+    effect->data.miscParticles->pos.x = posX;
+    effect->data.miscParticles->pos.y = posY;
+    effect->data.miscParticles->pos.z = posZ;
+    return ApiStatus_DONE2;
+}
+
```

Comment:
nit: use 1 empty line between blocks

Reply:
An empty line before the return, you mean? Should I do that for every func then?